graphql_compiler/compiler/emit_sql.py

```diff
@@ -259,23 +271,19 @@ def _get_xml_path_clause(output_column, predicate_expression):
     )
 
 
-# 3-tuple describing the join information for each traversal in a fold.
-#
-# Contains DirectJoinDescriptor naming the columns used in the join predicate,
-# the source/from table, and the destination/to table
-SQLFoldTraversalDescriptor = namedtuple(
-    "SQLFoldJoinInfo",
-    (
-        # DirectJoinDescriptor giving columns used to join from_table/to_table
-        "join_descriptor",
-        # SQLAlchemy table corresponding to corresponding to the outside vertex of the traversal,
-        # appears on the left side of the join.
-        "from_table",
-        # SQLAlchemy table corresponding to corresponding to the inside vertex of the traversal,
-        # appears on the right side of the join.
-        "to_table",
-    ),
-)
+class SQLFoldTraversalDescriptor(NamedTuple):
```

Comment:
(this is fine) In principle, we could and probably should move away over time from `NamedTuple` and toward `dataclass`, since dataclasses are more dev-friendly, with essentially everything `NamedTuple` has + automatic repr methods etc.